app/src/main/java/de/danoeh/antennapod/ui/episodeslist/FeedItemMenuHandler.java

```diff
@@ -194,7 +194,7 @@ public static boolean onMenuItemClicked(@NonNull Fragment fragment, int menuItem
                 PlaybackPreferences.writeNoMediaPlaying();
                 IntentUtils.sendLocalBroadcast(context, PlaybackServiceInterface.ACTION_SHUTDOWN_PLAYBACK_SERVICE);
             }
-            DBWriter.markItemPlayed(selectedItem, FeedItem.UNPLAYED, true);
+            DBWriter.markItemPlayed(FeedItem.UNPLAYED, selectedItem);
```

Comment:
This branch is supposed to reset the position. I think now it doesn't anymore because the called function only resets when marking as played. So it breaks the "reset playback position" long-press action